### PR TITLE
adds confidence distribution chart

### PR DESCRIPTION
**Pull request by jhoetter**
use [gateway](https://github.com/code-kern-ai/refinery-gateway/pull/45) and [submodule](https://github.com/code-kern-ai/refinery-submodule-model/pull/10) from this branch.

When you jump into the clickbait project, you should see a confidence distribution chart on the overview page

### CODE REVIEW

File: src/app/monitor/components/confidence-line-chart/confidence-line-chart.component.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<kern-line-chart [data]="dataInput">
+</kern-line-chart>
```
Comment (JWittmeyer) [inline src/app/monitor/components/confidence-line-chart/confidence-line-chart.component.html:2]: why is this wrapped in another component? i know we have it for other charts as well but it doesn't make sense to me

File: src/app/project-overview/components/project-overview/project-overview.component.html [hunk 2]
```diff
@@ -219,6 +219,22 @@
             </div>
         </div>
 
+        <div class="mt-8 w-2/4 grid w-full"
+            [ngClass]="displayGraphsForm.value == DisplayGraphsType.ALL || displayGraphsForm.value == DisplayGraphsType.LABEL_DISTRIBUTION?null:'hidden'">
```
Comment (JWittmeyer) [inline src/app/project-overview/components/project-overview/project-overview.component.html]: is this meant to be visible with the LABEL_DISTRIBUTION if you select it or is another state missing here?

File: src/app/monitor/components/confidence-line-chart/confidence-line-chart.component.html (hunk 1 above)
Comment (jhoetter) [reply src/app/monitor/components/confidence-line-chart/confidence-line-chart.component.html:2]: I only got it working that way tbh haha. Thought this was required for the modules.

File: src/app/project-overview/components/project-overview/project-overview.component.html (hunk 2 above)
Comment (jhoetter) [reply src/app/project-overview/components/project-overview/project-overview.component.html]: no, you're right, i missed that. will add it

**Comment by JWittmeyer** [comment]
Something seems to be up with the percentile display:
image
this happens when I use the no sample number

- [x] resolved

**Comment by jhoetter** [comment]
discussed the point re 100 samples, so I resolved it :)

**Comment by FelixKirsch** [comment]
In some settings, taking the first 100 records for the confidence chart results in a bad representation of the project. I played around with the conversational-ai project and deleted one heuristic after another and triggered new weak supervision runs. At some point, the confidence chart showed around 100% confidence for all records, while there were still a significant amount of labels with lower weak supervision confidence. I send you the project export.

**Comment by JWittmeyer** [comment]
if the selected amount is < 100 (e.g. with a slice filter) this will resulting < 100 percentil

- [x] resolved

**Review by JWittmeyer — APPROVED** [review]

**Pull request merged**